### PR TITLE
Draft features

### PR DESCRIPTION
**Pull request by Gnwin**
All recommended changes have been effected.
We replaced icons with images.
Awaiting approval.

### CODE REVIEW

**Comment by santiago-dev96** [comment]
# PR under review by @santiago-rodrig (TSE)

File: index.html [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+            </nav>
+           
+            <form class="search">
+                <input type="text" placeholder="Search" value=""/>
+                <button><i class="fa fa-search"></i></button>
+            </form>
+
+            <img class="upload" src="/assets/images/logos/upload.png" alt="upload">
+            <img class="apps" src="/assets/images/logos/apps.png" alt="apps">
+            <img class="user" src="/assets/images/logos/user.png" alt="user">
```
Comment (santiago-dev96) [inline index.html]: Images are not being displayed, you should use relative paths.
```suggestion
            <img class="upload" src="./assets/images/logos/upload.png" alt="upload">
            <img class="apps" src="./assets/images/logos/apps.png" alt="apps">
            <img class="user" src="./assets/images/logos/user.png" alt="user"> 
```

**Review by santiago-dev96 — CHANGES_REQUESTED** [review]
# Changes required 🔧

Your project is getting closer to an approvable project.

1. The width of the content should be 1024 pixels, use your browser developer tools to inspect the page you want to clone (right-click -> inspect)
2. Your images are not being displayed, you are using absolute paths, that is not working because that path will be from the hard drive's root and not from the HTML file you are using, use relative paths if you want the path to be relative to the HTML file (I left a comment in your code)

After you have implemented the required changes submit another CRR.

You can find me in slack as **srodrig**, happy coding! :wave:

File: index.html (hunk 1 above)
Comment (Gnwin) [reply index.html]: changes effected. awaiting approval

**Pull request closed without merging**